.github/workflows/authentication.yml

```diff
@@ -27,27 +30,26 @@ env:
   secrets_passphrase: ${{ secrets.GHASECRETSGPGPASSPHRASE1 }}
 
 jobs:
-  cocoapods:
-    name: cocoapods
+  spm:
     runs-on: macOS-15
     env:
-      SPM: false
-      LEGACY: false
       OS: iOS
+      SETUP: authentication
+      SPM: true
+      LEGACY: false
+      DIR: authentication
       DEVICE: iPhone 16
       TEST: false
+      SCHEME: AuthenticationExample
     steps:
       - name: Checkout
         uses: actions/checkout@master
+      - name: Xcode
+        run: sudo xcode-select -s /Applications/Xcode_16.4.app/Contents/Developer
```

Comment:
Should the xcode-select be migrated to one of the common scripts?

Reply:
Definitely. Will do in future PR.